### ISSUE TITLE
[Android] Bug/Error when using selfManaged mode.

### ISSUE THREAD

**Issue by anwersolangi** [react-native-webrtc/react-native-callkeep#424]
# Bug report
I have encountered a bug, if **selfManaged** mode is set to true and use **displayIncomingCall** without **number** parameter, then a **error** occur with no information of error, except ssp.
On further debugging, I see that there is a function **fromParts** at line 189 in **`RNCallKeepModule`**.java, code:
`Uri uri = Uri.fromParts(PhoneAccount.SCHEME_TEL, number, null);`
Is it a bug or something necessary to use connectionservice?
- [x] I've checked the [example](https://github.com/react-native-webrtc/react-native-callkeep/tree/master/example) to reproduce the issue.

- Reproduced on:
 - [x] Android
 - [ ] iOS

## Steps to Reproduce
1. Create new project, install react-native-callkeep.
2. set selfManaged true, and use RNCallKeep.displayIncomingCall(uuid(),null,notification.data?.username,null,true);

## Versions
    - Callkeep: 4.2.0
    - React Native: 0.63.4
    - iOS:
    - Android: 10
    - Phone model: Moto One Macro

**Comment by manuquentin** [comment]
Hi @anwersolangi thanks for the report.

Yeah I don't think that the handle is required in self managed, can you try with line 196 and 198 commented out in `android/src/main/java/io/wazo/callkeep/RNCallKeepModule.java` ?

Meanwhile you can try with :
```
RNCallKeep.displayIncomingCall(uuid(),'',notification.data?.username,null,true);
```
Using an empty string instead of `null` as the number parameter.

**Comment by anwersolangi** [comment]
> Hi @anwersolangi thanks for the report.
> 
> Yeah I don't think that the handle is required in self managed, can you try with line 196 and 198 commented out in `android/src/main/java/io/wazo/callkeep/RNCallKeepModule.java` ?
> 
> Meanwhile you can try with :
> 
> ```
> RNCallKeep.displayIncomingCall(uuid(),'',notification.data?.username,null,true);
> ```
> 
> Using an empty string instead of `null` as the number parameter.

Yeah i have already tried it with empty string, and if we use empty string, then the notification of incomingcall is only shown if the system phone app is opened, or if someone is calling us, but the event showIncomingCallUi is fired.

**Comment by agam-colaburate** [comment]
+1

**Comment by agam-colaburate** [comment]
@manuquentin @anwersolangi  
Any lead on this ?

**Comment by namnm** [comment]
Why dont we just put something instead of null/empty string?
Like `Unknown` or `App Name`?
